assets/translations.json

```diff
@@ -110,7 +110,7 @@
         "plotTitle": "Image count over time ({{windowLengthInDays}}-day intervals)",
         "inspect": {
           "station": "Station {{id}}",
-          "observations": "{{count}} observations",
+          "observations": "{{current}}{{count}} observations",
```

Comment:
The first solution is not working because {{count}} unfortunately accepts only numbers, not strings. So I added a separate fields in translations.json file.